Update README.md to describe top N articles endpoint

mrober01 opened the pull request.

SomeOddCodeGuy commented:
Could you peek at your last line, stating the default values? I believe that the default value for num_articles is 8, but in the readme you listed it as 80.

```
async def get_top_n_full_articles_by_prompt(
    prompt: str = Query(..., description="Search prompt"),
    percentile: float = Query(0.5, description="Percentile for search relevance"),
    num_results: int = Query(20, description="Number of results to return"),
    num_top_articles: int = Query(8, description="number of top articles to return")
```

SomeOddCodeGuy commented:
I can fix that easily after completing this, I just want to be sure which you found to be better- 80 or 8. If 80, you might need to update this PR to go update that in the code. I finished the other PR already.

So this is less about nit-picking and more about figuring out which you wanted it to be.

mrober01 commented:
Yes definitely top_n_results is 8 not 80. 80 would be too much.  You can update the readme.  Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Chris ***@***.***>
Sent: Sunday, January 12, 2025 5:13:27 PM
To: SomeOddCodeGuy/OfflineWikipediaTextApi ***@***.***>
Cc: mrober01 ***@***.***>; Author ***@***.***>
Subject: Re: [SomeOddCodeGuy/OfflineWikipediaTextApi] Update README.md to describe top N articles endpoint (PR #6)


I can fix that easily after completing this, I just want to be sure which you found to be better- 80 or 8. If 80, you might need to update this PR to go update that in the code. I finished the other PR already.

So this is less about nit-picking and more about figuring out which you wanted it to be.

—
Reply to this email directly, view it on GitHub<https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi/pull/6#issuecomment-2586015932>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADS62ON3SO5UEP6WPQ3VG632KMHLPAVCNFSM6AAAAABVAWL5IOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBWGAYTKOJTGI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

mrober01 reviewed (commented):
Yes, i meant 8, not 80 for default N .

SomeOddCodeGuy commented:
Sounds good. Im going to wait to merge this same as Wilmer until your account thing is figured out!

mrober01 commented:
Hi,
I think i figured it out.  On my linux machine, I did not have my git email address set and apparently that can cause this.  I updated it and submitted two more minor commits where I updated comments, those i think should track properly.  But interestingly enough, when I updated the Readme file, I did it directly from github, and the commit does seem to have my email address - so, I think those pull requests would track properly.

Thanks,
Mike

________________________________
From: Chris ***@***.***>
Sent: Sunday, January 12, 2025 6:30 PM
To: SomeOddCodeGuy/OfflineWikipediaTextApi ***@***.***>
Cc: mrober01 ***@***.***>; Author ***@***.***>
Subject: Re: [SomeOddCodeGuy/OfflineWikipediaTextApi] Update README.md to describe top N articles endpoint (PR #6)


Sounds good. Im going to wait to merge this same as Wilmer until your account thing is figured out!

—
Reply to this email directly, view it on GitHub<https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi/pull/6#issuecomment-2586066866>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADS62OMPMY63T44QZV4RY3T2KMQNBAVCNFSM6AAAAABVAWL5IOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOBWGA3DMOBWGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
